purchases/src/main/kotlin/com/revenuecat/purchases/common/AppConfig.kt

```diff
@@ -34,6 +35,13 @@ internal class AppConfig(
     var forceSigningErrors: Boolean = forceSigningErrors
         get() = runningTests && field
 
+    private val _isAppBackgrounded: AtomicBoolean = AtomicBoolean(true)
```

Comment:
Just to confirm. Is the initial value of `true` correct? In other words, does the system calls `onAppForegrounded()` when launching the app?

Reply:
Makes sense. I was sure you had thought about this, but wanted to double check. Thanks for the explanation!